src/common/gateway/entities/transaction.inner.ts

```diff
@@ -0,0 +1,17 @@
+export class TransactionInner {
+  constructor(init?: Partial<TransactionInner>) {
+    Object.assign(this, init);
+  }
+
+  nonce: number = 0;
+  value: string = '';
+  receiver: string = '';
+  sender: string = '';
+  gasPrice: number = 0;
+  gasLimit: number = 0;
+  data: string = '';
+  signature: string = '';
+  chainId: number = 0;
```

Comment:
Perhaps `chainID` to be consistent with the Proxy / Gateway?

https://devnet-gateway.multiversx.com/transaction/c01cae88fb9c9fd6f7a82406fd63c36f186953d3436dcce13d7651ed89a4c893?withResults=true

Additionally, it should be a string (not a number).